### PR TITLE
Matt Toensing 😁

### PR DESCRIPTION
**Pull request by matttoensing**
Almost made it through iteration 4, I could not figure out the last trailing zero!

### CODE REVIEW

File: lib/doctor.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+class Doctor
+  attr_reader :name, :specialty, :education, :salary
+
+  def initialize(doctor_info)
+    @name = doctor_info[:name]
+    @specialty = doctor_info[:specialty]
+    @education = doctor_info[:education]
+    @salary = doctor_info[:salary]
+  end
```
Comment (mikedao) [inline lib/doctor.rb:10]: Great

File: lib/network.rb [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  def initialize(name)
+    @name = name
+    @hospitals = []
+  end
+
+  def add_hospital(hospital)
+    @hospitals << hospital
+  end
+
+  def highest_paid_doctor
```
Comment (mikedao) [inline lib/network.rb:14]: This only works if the highest paid doctor is in the last hospital that is added to the network.

**Review by mikedao — COMMENTED** [review]
Good job throughout, just keep in mind other situations that are plausible that may cause your code to not work as intended.

**Comment by matttoensing** [comment]
Thank you Mike! I really appreciate the feedback

**Pull request closed without merging**